editoast/src/main.rs

```diff
@@ -68,6 +68,7 @@ pub fn create_server(
         .merge(("port", runserver_config.port))
         .merge(("address", runserver_config.address.clone()))
         .merge(("databases.postgres.url", pg_config.url()))
+        .merge(("databases.postgres.timeout", 10))// Timeout needed for MacOS M1 & M2 
```

Comment:
Is it really usefull ?

```suggestion
        .merge(("databases.postgres.timeout", 10)) // Timeout needed for MacOS M1 & M2 
```